fedgov-cv-ISDcreator-webapp/src/main/webapp/js/depositor.js

```diff
@@ -310,6 +312,15 @@ function createMessageJSON()
             "speedLimit": box.features[i].attributes.speedLimit,
             "drivingLanes": drivingLaneArray
         };
+        
+        if (approachArray[i].approachType === undefined) {
+            incompleteApproaches.push(lanes.features[i].attributes.laneNumber)
+
+            if (incompleteApproaches) {

```

Comment:
I guess you can change to check if the length of this array is great than 0 or I think you don't need to check this here as you are already checking for undefined condition